kickstarter/android-oss
# NT-1648: Interstitial Screen UI

# 📲 What

- ActivityExtension for showing snackBarError
- ActivityExtension for showing snackbarSuccess
- Added new colors
- SnackBar extension to personalize backgroundColor and textColor
- Break down in showSnackbar output in EmailVerification viewModel into two, showSnackbarError and showSnackbarSuccess
- Dismiss keyboard when fragment pressented

# 👀 See
- **Success snackbar**
![Screen Shot 2020-11-27 at 10 38 32 AM](https://user-images.githubusercontent.com/4083656/100477756-2b0a6680-309e-11eb-93f5-5efbe54dc8ff.png)

- **Error snackbar**
![Screen Shot 2020-11-27 at 10 37 57 AM](https://user-images.githubusercontent.com/4083656/100477774-378ebf00-309e-11eb-8712-a01389d2a036.png)

- **Designs** 

 [Design – confirmation](https://www.figma.com/file/GMONc3QmlDx6rDCTHe56LJ/Phase-1?node-id=1671%3A363)
 [Design – error](https://www.figma.com/file/GMONc3QmlDx6rDCTHe56LJ/Phase-1?node-id=1831%3A465)

|  |  |

# 📋 QA

- Open interstitial screen on internal tools, without being logged in, press the re-send email button, you'll see the error snackbar.
- Open interstitial screen on internal tools, logged in, press the re-send email button, you'll see the success snackbar.

# Story 📖

[NT-1648](https://kickstarter.atlassian.net/browse/NT-1648)
